src/huggingface_hub/utils/_errors.py

```diff
@@ -46,50 +51,67 @@ class EntryNotFoundError(HTTPError):
     ```
     """
 
+    def __init__(self, message, response):
+        super(EntryNotFoundError, self).__init__(message, response=response)
```

Comment:
Not necessary since Python 3!
```suggestion
        super().__init__(message, response=response)
```